tasmota/tasmota_template_ESP32.h

```diff
@@ -138,7 +138,7 @@ enum ProgramSelectablePins {
 // Text in webpage Module Parameters and commands GPIOS and GPIO
 const char kSensorNames[] PROGMEM =
   D_SENSOR_NONE "|"
-  D_SENSOR_BUTTON "|" D_SENSOR_BUTTON "_n|" D_SENSOR_BUTTON "_i|" D_SENSOR_BUTTON "_in|"
+  D_SENSOR_BUTTON "|" D_SENSOR_BUTTON "_n|" D_SENSOR_BUTTON "_i|" D_SENSOR_BUTTON "_in|" D_SENSOR_BUTTON "_tc|"
```

Comment:
See previous comment

Reply:
Yes I know, but I considered the ESP32 still as alpha, which would give us the chance to reorganize some (tiny) things, knowing that something will get broken. On the ESP8266 I would not have suggested such a thing.

Shall I still change it?

Reply:
Yes pls change as it breaks current webcam configs.

Reply:
Okay, I will make the changes.